clients.json

```diff
@@ -251,6 +251,7 @@
   {
     "name": "Redisson",
     "language": "Java",
+    "url": "https://redisson.org/",
     "repository": "https://github.com/mrniko/redisson",
     "description": "distributed and scalable Java data structures on top of Redis server",
     "authors": ["mrniko"],
```

Comment:
On redisson.org, the github link points to https://github.com/redisson/redisson. Should the "repository" field be updated here too?